Add various design improvements

PeeterPaal opened the pull request:
- Use `object.og_image` method for OG image
- Add titles & tooltips to various content areas

Closes #21 
Closes #22

PeeterPaal committed:
Use obj.og_image method for OG image

PeeterPaal committed:
Add titles & tooltips to various content areas

tanelj commented on components/site-footer.tpl:
```diff
@@ -51,7 +51,11 @@
+          <div class="footer-contact content-area">
+            {%- assign footer_content_title = "footer" | lce -%}
+            {%- assign footer_content_title_tooltip = "content_tooltip_all_pages_same_language" | lce -%}
+            {% xcontent name="footer" title_tooltip=footer_content_title_tooltip %}
```
`footer_content_title` is missing.
```suggestion
            {% xcontent name="footer" title=footer_content_title title_tooltip=footer_content_title_tooltip %}
```

tanelj reviewed (changes requested):
Nice, but one title is missing.

PeeterPaal committed:
Improve site footer

Add min-width to site footer to make it easier to locate and add
title to the content area.

pxska reviewed (approved):
Looks good!

tanelj reviewed (approved):
Nice!
